website/docs/dashboard/dashboard_user_guide.md

```diff
@@ -128,6 +128,13 @@ The settings you can set on this page are:
 - `m12_on` (motor C on)
 - `m13_on` (motor D on)
 
+:::info
+You should press the physical E-stop prior to turning the motors on / off
+(with `m10_on`, `m11_on`, `m12_on`, `m13_on`).
+
+If you do not press the physical E-stop, the wheels will brake until it is pressed -> released.
```

Comment:
We should consider making this a warning to help.

Reply:
Changing to a caution